lang/ui.en.json

```diff
@@ -247,6 +247,10 @@
     "defaultMessage": "Don't show this again",
     "description": "Text to never show a dialog again"
   },
+  "download-action": {
+    "defaultMessage": "Download",
+    "description": "Download action label"
+  },
```

Comment:
Not used.

Reply:
Thanks!

Reply:
aefa21dd3572f9fa2fa605b70ff5831e959acdb7